ci/L0_backend_vllm/accuracy_test/test.sh

```diff
@@ -44,6 +45,13 @@ sed -i 's/"gpu_memory_utilization": 0.5/"gpu_memory_utilization": 0.3/' models/v
 
 RET=0
 
+set +e
+# Need to generate baseline first, since running 2 vLLM engines causes
+# memory issues: https://github.com/vllm-project/vllm/issues/2248
+python -c "import accuracy_test; accuracy_test.prepare_vllm_engine_outputs()" >> $VLLM_ENGINE_LOG 2>&1 & BASELINE_PID=$!
```

Comment:
Nit: I think it's a bit confusing to have us call the script `accuracy_test.prepare_vllm_engine_outputs()` and then use `CLIENT_PY`. Can we generalize a bit here?

Something like 
```
python3 $CLIENT_PY --generate-baseline
```
and argparse on the script?

Reply:
Nice idea, thanks! Will incorporate

Reply:
revised